examples/docker/kafka-oauth-strimzi/kafka/start.sh

```diff
@@ -14,4 +14,7 @@ wait_for_url "$URI/realms/${REALM:-demo}" "Waiting for realm '${REALM}' to be av
 
 ./simple_kafka_config.sh | tee /tmp/strimzi.properties
 
+# add extra jars to classpath
```

Comment:
Got to be worth including in the comment which extra jars this is intended to add

Reply:
Something like the following?

`# add Strimzi kafka-oauth-* jars and its dependencies to classpath`

Reply:
:+1:

Reply:
@tombentley Fixed